src/Uno.UI.RuntimeTests/Tests/Microsoft_UI_Xaml_Controls/Given_TreeView.cs

```diff
@@ -318,6 +318,86 @@ public async Task When_TreeViewItem_Dragged_NRE()
 
 			Assert.IsFalse(TestableTreeViewItem.DraggingThrewException);
 		}
+
+		[TestMethod]
+#if !HAS_INPUT_INJECTOR
+		[Ignore("InputInjector is only supported on skia")]
+#elif HAS_UNO && !HAS_UNO_WINUI
+		[Ignore("Fails on UWP branch as mixing WUX and MUX types causes errors.")]
```

Comment:
I think it should be testable on UWP tree. What kind of error do you get?

Reply:
Not sure, it was a long time ago (this is a copy from the test above it). It's most likely a mistake in the test itself, but it seems like it would be a waste of time to find where the problem is exactly.